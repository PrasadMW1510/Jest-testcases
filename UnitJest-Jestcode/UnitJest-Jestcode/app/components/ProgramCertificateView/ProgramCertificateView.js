/**
 *
 * ProgramCertificateView
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
import ProgramCertificateList from 'components/ProgramCertificateList';

import './ProgramCertificateView.scss';

export class ProgramCertificateView extends React.Component {
  renderCertificateList = list => {
    if (list && list.length > 0) {
      return list.map(item => (
        <option key={item.certificate_id[0]} value={item.certificate_id[0]}>
          {item.certificate_name[0]}
        </option>
      ));
    }
    return null;
  };

  renderCertificates = certificateInfo => {
    if (certificateInfo && !Array.isArray(certificateInfo)) {
      return this.renderCertificateList(certificateInfo.certificates[0].certificate);
    }
    return null;
  };

  render() {
    const {
      certificateInfo,
      selectedCertificate,
      lineOne,
      lineTwo,
      lineThree,
      lineFour,
      selectedClasses,
      toggleRowSelection,
      toggleRowsSelection,
    } = this.props;
    return (
      <div className="certinfo">
        <span className="certinfo__header">Certificate Manager</span>
        <div className="awardselect">
          <span className="awardselect__header"> Select a Certificate </span>
          <div className="awardselect__cont">
            <span className="awardselect__label">Select a program certificate:</span>
            <select
              className="awardselectbox"
              name="selectedCertificate"
              onChange={this.props.handleChange}
              value={selectedCertificate}
            >
              <option> -- Certificate Name -- </option>
              {this.renderCertificates(certificateInfo)}
            </select>
          </div>
          <div className="messagecont">
            <span className="messagecont__header">Enter a custom message:</span>
            <ul className="customlines">
              <li className="customlines__item">
                <span className="customlines__text">Line 1:</span>
                <input
                  className="customlines__input"
                  id="line1"
                  type="text"
                  autoComplete="off"
                  name="lineOne"
                  value={lineOne}
                  onChange={this.props.handleChange}
                />
              </li>
              <li className="customlines__item">
                <span className="customlines__text">Line 2:</span>
                <input
                  className="customlines__input"
                  id="line2"
                  type="text"
                  autoComplete="off"
                  name="lineTwo"
                  value={lineTwo}
                  onChange={this.props.handleChange}
                />
              </li>
              <li className="customlines__item">
                <span className="customlines__text">Line 3:</span>
                <input
                  className="customlines__input"
                  id="line3"
                  type="text"
                  autoComplete="off"
                  name="lineThree"
                  value={lineThree}
                  onChange={this.props.handleChange}
                />
              </li>
              <li className="customlines__item">
                <span className="customlines__text">Line 4:</span>
                <input
                  className="customlines__input"
                  id="line4"
                  type="text"
                  autoComplete="off"
                  name="lineFour"
                  value={lineFour}
                  onChange={this.props.handleChange}
                />
              </li>
            </ul>
          </div>
        </div>
        <ProgramCertificateList
          certificateInfo={certificateInfo}
          selectedCertificate={this.props.selectedCertificate}
          toggleRowSelection={toggleRowSelection}
          selectedClasses={selectedClasses}
          toggleRowsSelection={toggleRowsSelection}
        />
      </div>
    );
  }
}

ProgramCertificateView.propTypes = {
  certificateInfo: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
  handleChange: PropTypes.func,
  selectedCertificate: PropTypes.string,
  lineOne: PropTypes.string,
  lineTwo: PropTypes.string,
  lineThree: PropTypes.string,
  lineFour: PropTypes.string,
  selectedClasses: PropTypes.array,
  toggleRowSelection: PropTypes.func,
  toggleRowsSelection: PropTypes.func,
};

export default ProgramCertificateView;
