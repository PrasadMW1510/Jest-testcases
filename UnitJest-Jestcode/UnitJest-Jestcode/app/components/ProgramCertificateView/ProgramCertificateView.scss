.awardselect {
  background-color: #fbe380;
  height: 200px;
  border: 1px solid #000;
  font-size: 11px;
  margin-top: 8px;
}

.awardselect__header {
  background-color: #f4cc37;
  display: block;
  font-size: 11px;
  font-weight: bold;
  padding: 5px 5px 5px 15px;
}

.certinfo {
  padding-top: 8px;
}

.certinfo__header {
  font-size: 14px;
  font-weight: bold;
}

.awardselect__cont {
  margin-top: 10px;
  padding-left: 15px;
}

.awardselectbox {
  margin-left: 50px;
  background-color: #fff;
  width: 200px;
  border: 1px solid #000;
  appearance: none;
  border-radius: 0;
  padding-left: 5px;
}

.messagecont {
  margin-top: 20px;
  margin-left: 15px;
}

.customlines {
  margin-left: 65px;
  margin-top: 5px;
  list-style-type: none;
}

.customlines__input {
  background-color: #fff;
  margin-left: 40px;
  width: 150px;
  border: 1px solid #000;
}

.customlines__item {
  padding-bottom: 10px;
}
