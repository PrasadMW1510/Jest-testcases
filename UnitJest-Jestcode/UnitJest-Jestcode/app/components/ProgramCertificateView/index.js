/**
 *
 * ProgramCertificateView
 *
 */

export { default } from './ProgramCertificateView';
