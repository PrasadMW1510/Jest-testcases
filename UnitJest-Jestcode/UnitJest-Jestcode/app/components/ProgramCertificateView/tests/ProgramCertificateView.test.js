import React from 'react';

import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ProgramCertificateView from '../ProgramCertificateView';

describe('ProgramCertificateView  should return no data', () => {
  let wrapper = null;
  let certificateInfo = null;
  beforeEach(() => {
    certificateInfo = {
      certificates: [
        {
          certificate: {},
        },
      ],
    };
    wrapper = shallow(<ProgramCertificateView certificateInfo={certificateInfo} />);
  });

  it('Should render as expected', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});

describe('ProgramCertificateView  should return valid certificateInfo', () => {
  let wrapper = null;
  let certificateInfo = null;
  beforeEach(() => {
    certificateInfo = {
      certificates: [
        {
          certificate: [
            {
              certificate_id: 'fmng_11',
              certificate_name: 'FASTT Math NG Award Certificate',
              certificate_type: 'Default',
              parent_app_id: 'FMNG',
            },
            {
              certificate_id: 'fm_11',
              certificate_name: 'FASTT Math Award Certificate',
              certificate_type: 'Default',
              parent_app_id: 'FM',
            },
          ],
        },
      ],
      recipients: {
        recipient: [
          {
            name: 'DoTheMath',
            id: 'sc49dbt98j3dbigsb1e73f31_2efa7f0',
            type: 'class',
            grade: '5',
          },
          {
            name: 'E3D CA V1',
            id: 'fs8lgl4fohettf9h1401pb56_2efa7f0',
            type: 'class',
            grade: 'K',
          },
          {
            name: 'E3D CA V2',
            id: 'mff03ugu34i2lrrmt0j39vlg_2efa7f0',
            type: 'class',
            grade: '5',
          },
          {
            name: 'E3D CB V1',
            id: 'c8eib35pi81ada20bp0pmltg_2efa7f0',
            type: 'class',
            grade: 'PK',
          },
        ],
      },
    };
    wrapper = shallow(<ProgramCertificateView certificateInfo={certificateInfo} />);
  });

  it('Should render as expected', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('Component Will recieve Props', () => {
    wrapper.setProps({ selectedCertificate: 'fmng11' });
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('Component Will not recieve Props if its an array', () => {
    wrapper.setProps({ selectedCertificate: 'fmng11' });
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('handleChange', () => {
    wrapper.find('.awardselectbox').simulate('change', {
      target: { value: 'fm_11' },
    });
    expect(wrapper.find('option')).toHaveLength(3);
  });
});

describe('ProgramCertificateView , if an array , pass no Data', () => {
  let wrapper = null;
  let certificateInfo = null;
  beforeEach(() => {
    certificateInfo = [];
    wrapper = shallow(<ProgramCertificateView certificateInfo={certificateInfo} />);
  });

  it('Should render as expected', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});

describe('Component Will not receive Props', () => {
  let wrapper = null;
  let certificateInfo = null;
  beforeEach(() => {
    certificateInfo = [];
    wrapper = shallow(<ProgramCertificateView certificateInfo={certificateInfo} />);
  });

  it('Component Will not receive Props', () => {
    wrapper.setProps({ selectedCertificate: 'fmng100' });
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
