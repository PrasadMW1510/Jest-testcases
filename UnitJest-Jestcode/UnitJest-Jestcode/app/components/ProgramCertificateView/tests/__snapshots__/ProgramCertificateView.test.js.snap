// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Will not receive Props Component Will not receive Props 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
        value="fmng100"
      >
        <option>
           -- Certificate Name -- 
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={Array []}
    selectedCertificate="fmng100"
  />
</div>
`;

exports[`ProgramCertificateView  should return no data Should render as expected 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
      >
        <option>
           -- Certificate Name -- 
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={
      Object {
        "certificates": Array [
          Object {
            "certificate": Object {},
          },
        ],
      }
    }
  />
</div>
`;

exports[`ProgramCertificateView  should return valid certificateInfo Component Will not recieve Props if its an array 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
        value="fmng11"
      >
        <option>
           -- Certificate Name -- 
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={
      Object {
        "certificates": Array [
          Object {
            "certificate": Array [
              Object {
                "certificate_id": "fmng_11",
                "certificate_name": "FASTT Math NG Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FMNG",
              },
              Object {
                "certificate_id": "fm_11",
                "certificate_name": "FASTT Math Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FM",
              },
            ],
          },
        ],
        "recipients": Object {
          "recipient": Array [
            Object {
              "grade": "5",
              "id": "sc49dbt98j3dbigsb1e73f31_2efa7f0",
              "name": "DoTheMath",
              "type": "class",
            },
            Object {
              "grade": "K",
              "id": "fs8lgl4fohettf9h1401pb56_2efa7f0",
              "name": "E3D CA V1",
              "type": "class",
            },
            Object {
              "grade": "5",
              "id": "mff03ugu34i2lrrmt0j39vlg_2efa7f0",
              "name": "E3D CA V2",
              "type": "class",
            },
            Object {
              "grade": "PK",
              "id": "c8eib35pi81ada20bp0pmltg_2efa7f0",
              "name": "E3D CB V1",
              "type": "class",
            },
          ],
        },
      }
    }
    selectedCertificate="fmng11"
  />
</div>
`;

exports[`ProgramCertificateView  should return valid certificateInfo Component Will recieve Props 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
        value="fmng11"
      >
        <option>
           -- Certificate Name -- 
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={
      Object {
        "certificates": Array [
          Object {
            "certificate": Array [
              Object {
                "certificate_id": "fmng_11",
                "certificate_name": "FASTT Math NG Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FMNG",
              },
              Object {
                "certificate_id": "fm_11",
                "certificate_name": "FASTT Math Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FM",
              },
            ],
          },
        ],
        "recipients": Object {
          "recipient": Array [
            Object {
              "grade": "5",
              "id": "sc49dbt98j3dbigsb1e73f31_2efa7f0",
              "name": "DoTheMath",
              "type": "class",
            },
            Object {
              "grade": "K",
              "id": "fs8lgl4fohettf9h1401pb56_2efa7f0",
              "name": "E3D CA V1",
              "type": "class",
            },
            Object {
              "grade": "5",
              "id": "mff03ugu34i2lrrmt0j39vlg_2efa7f0",
              "name": "E3D CA V2",
              "type": "class",
            },
            Object {
              "grade": "PK",
              "id": "c8eib35pi81ada20bp0pmltg_2efa7f0",
              "name": "E3D CB V1",
              "type": "class",
            },
          ],
        },
      }
    }
    selectedCertificate="fmng11"
  />
</div>
`;

exports[`ProgramCertificateView  should return valid certificateInfo Should render as expected 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
      >
        <option>
           -- Certificate Name -- 
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
        <option
          key="f"
          value="f"
        >
          F
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={
      Object {
        "certificates": Array [
          Object {
            "certificate": Array [
              Object {
                "certificate_id": "fmng_11",
                "certificate_name": "FASTT Math NG Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FMNG",
              },
              Object {
                "certificate_id": "fm_11",
                "certificate_name": "FASTT Math Award Certificate",
                "certificate_type": "Default",
                "parent_app_id": "FM",
              },
            ],
          },
        ],
        "recipients": Object {
          "recipient": Array [
            Object {
              "grade": "5",
              "id": "sc49dbt98j3dbigsb1e73f31_2efa7f0",
              "name": "DoTheMath",
              "type": "class",
            },
            Object {
              "grade": "K",
              "id": "fs8lgl4fohettf9h1401pb56_2efa7f0",
              "name": "E3D CA V1",
              "type": "class",
            },
            Object {
              "grade": "5",
              "id": "mff03ugu34i2lrrmt0j39vlg_2efa7f0",
              "name": "E3D CA V2",
              "type": "class",
            },
            Object {
              "grade": "PK",
              "id": "c8eib35pi81ada20bp0pmltg_2efa7f0",
              "name": "E3D CB V1",
              "type": "class",
            },
          ],
        },
      }
    }
  />
</div>
`;

exports[`ProgramCertificateView , if an array , pass no Data Should render as expected 1`] = `
<div
  className="certinfo"
>
  <span
    className="certinfo__header"
  >
    Certificate Manager
  </span>
  <div
    className="awardselect"
  >
    <span
      className="awardselect__header"
    >
       Select a Certificate 
    </span>
    <div
      className="awardselect__cont"
    >
      <span
        className="awardselect__label"
      >
        Select a program certificate:
      </span>
      <select
        className="awardselectbox"
        name="selectedCertificate"
      >
        <option>
           -- Certificate Name -- 
        </option>
      </select>
    </div>
    <div
      className="messagecont"
    >
      <span
        className="messagecont__header"
      >
        Enter a custom message:
      </span>
      <ul
        className="customlines"
      >
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 1:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line1"
            name="lineOne"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 2:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line2"
            name="lineTwo"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 3:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line3"
            name="lineThree"
            type="text"
          />
        </li>
        <li
          className="customlines__item"
        >
          <span
            className="customlines__text"
          >
            Line 4:
          </span>
          <input
            autoComplete="off"
            className="customlines__input"
            id="line4"
            name="lineFour"
            type="text"
          />
        </li>
      </ul>
    </div>
  </div>
  <ProgramCertificateList
    certificateInfo={Array []}
  />
</div>
`;
